src/components/DocumentationTopic/ContentTableSection.vue

```diff
@@ -24,14 +24,21 @@
 </template>
 
 <script>
+import SectionTitle from 'docc-render/components/ContentNode/SectionTitle.vue';
+import { anchorize } from 'docc-render/utils/strings';
+
 export default {
   name: 'ContentTableSection',
+  components: { SectionTitle },
   props: {
     title: {
       type: String,
       required: true,
     },
   },
+  computed: {
+    anchor: ({ title }) => anchorize(title),
```

Comment:
I know that DocC is not currently providing `anchor` values for these headings, but I think that work is being tracked since we ideally want it to be in control of the id as opposed to having it being generated it here—I think we could utilize `anchorize` as a fallback in the meantime, but we may run into future issues where old links might not work as one system takes over.

Reply:
I added most of the logic from https://github.com/apple/swift-docc-render/pull/190 into this PR to address this

Reply:
Good idea, thanks. There could be future issues when the switch happens where the anchors currently being generated in the app may conflict with futures ones provided by DocC, but I think that on-page link URLs are generally pretty unstable with titles changing anyways, so it probably makes sense to rely on DocC and only generate them when they aren't for now.